src/wrappers/themis/jsthemis/secure_cell_context_imprint.cpp

```diff
@@ -42,6 +43,21 @@ namespace jsthemis {
 
   void SecureCellContextImprint::New(const Nan::FunctionCallbackInfo<v8::Value>& args) {
     if (args.IsConstructCall()) {
+      if(args.Length()<1){
+        ThrowError("Secure Cell (Context Imprint) constructor", THEMIS_INVALID_PARAMETER, "missing master key");
+        args.GetReturnValue().SetUndefined();
+        return;
+      }
+      if(!args[0]->IsUint8Array()){
+        ThrowError("Secure Cell (Context Imprint) constructor", THEMIS_INVALID_PARAMETER, "master key is not byte buffer");
```

Comment:
What do you think about suggesting a way to fix error?

```suggestion
        ThrowError("Secure Cell (Context Imprint) constructor", THEMIS_INVALID_PARAMETER, "master key is not a byte buffer, expect ByteBuffer or TypedArray");
```

if you like it, plz propagate to all similar errors

Reply:
I think that would be really helpful. Will do.